Code/Light.GuardClauses.Tests/ComparableAssertions/RangeTests.cs

```diff
@@ -49,4 +54,76 @@ public static void ConstructorException(int from, int to)
         act.Should().Throw<ArgumentOutOfRangeException>().And
            .Message.Should().Contain($"{nameof(to)} must not be less than {from}, but it actually is {to}.");
     }
+
+    [Theory]
+    [MemberData(nameof(Collections))]
+    public static void RangeForCollections(IEnumerable enumerable)
+    {
+        // ReSharper disable PossibleMultipleEnumeration
+        var range = Range.For(enumerable);
+
+        var expectedRange = new Range<int>(0, enumerable.Count(), true, false);
+        range.Should().Be(expectedRange);
+        // ReSharper restore PossibleMultipleEnumeration
+    }
+
+    public static readonly TheoryData<IEnumerable> Collections =
+        new ()
+        {
+            new List<int> { 1, 2 ,3 ,4},
```

Comment:
Nitpick: inconsistent spaces here.

Reply:
Solved in d20736c400556cbf174f7c195b08c69cc3ab6540